src/components/JsonSchemaValidator/Schemas/dbtNormalize_0.4.0.json

```diff
@@ -0,0 +1,240 @@
+      "minimum": 0,
+      "title": "Backfill Limit",
+      "group": "Operation and Logic",
+      "longDescription": "The maximum numbers of days of new data to be processed since the latest event processed. Please refer to the [incremental logic](/docs/modeling-your-data/modeling-your-data-with-dbt/package-mechanics/incremental-processing/#package-state) section for more details.",
+      "packageDefault": "30",
+      "description": "The maximum numbers of days of new data to be processed since the latest event processed"
+    },
+    "snowplow__days_late_allowed": {
+      "recommendFullRefresh": true,
+      "order": 42,
+      "consoleGroup": "advanced",
+      "type": "number",
+      "minimum": 0,
+      "title": "Days Late Allowed",
+      "group": "Operation and Logic",
+      "longDescription": "The maximum allowed number of days between the event creation and it being sent to the collector. Exists to reduce lengthy table scans that can occur as a result of late arriving data.",
+      "packageDefault": "3",
+      "description": "The maximum allowed number of days between the event creation and it being sent to the collector"
+    },
+    "snowplow__lookback_window_hours": {
+      "recommendFullRefresh": false,
+      "order": 43,
+      "consoleGroup": "advanced",
+      "type": "number",
+      "minimum": 0,
+      "title": "Event Lookback Window",
+      "longDescription": "The number of hours to look before the latest event processed - to account for late arriving data, which comes out of order.",
+      "packageDefault": "6",
+      "group": "Operation and Logic",
+      "description": "The number of hours to look before the latest event processed - to account for late arriving data, which comes out of order"
+    },
+    "snowplow__start_date": {
+      "recommendFullRefresh": false,
+      "order": 6,
+      "consoleGroup": "required",
+      "type": "string",
+      "format": "date",
+      "title": "Start Date",
+      "group": "Operation and Logic",
+      "longDescription": "The date to start processing events from in the package on first run or a full refresh, based on `collector_tstamp`",
+      "packageDefault": "2020-01-01",
+      "description": "The date to start processing events from in the package on first run or a full refresh, based on `collector_tstamp`"
+    },
+    "snowplow__session_timestamp": {
+      "recommendFullRefresh": false,
+      "order": 55,
+      "consoleGroup": "advanced",
+      "type": "string",
+      "title": "Start Date",
+      "group": "Operation and Logic",
+      "longDescription": "This determines which timestamp is used to process sessions of data. It's a good idea to have this timestamp be the same timestamp as the field you partition your events table on.",
+      "packageDefault": "collector_tstamp",
+      "description": "Determines which timestamp is used to process sessions of data"
+    },
+    "snowplow__partition_tstamp": {
+      "recommendFullRefresh": false,
```

Comment:
Are we sure we don't recommend a full refresh when changed?

Reply:
We could keep it non full refresh, but lets be safe.